Tasks/Maven/pmdForMaven.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference path='../../definitions/vsts-task-lib.d.ts' />
+
+import tl = require('vsts-task-lib/task');
+import trm = require('vsts-task-lib/toolrunner');
+
+// Adds PMD goals, if selected by the user
+export function applyPmdArgs(mvnRun: trm.ToolRunner):void {
```

Comment:
This file will also contain the parsing of XML output, so it does indeed grow with future PRs.